app/uk/gov/hmrc/apinotificationqueue/repository/NotificationRepository.scala

```diff
@@ -41,7 +42,7 @@ trait NotificationRepository {
 
```

Comment:
I think we may need a compound index on `clientId + datePulled` to support pulled/unpulled queries

Reply:
done